symfit/tests/tests_ode.py

```diff
@@ -105,8 +99,8 @@ def test_simple_kinetics(self):
         a0 = 54 * 10e-4
 
         model_dict = {
-            D(a, t): - k * a**2,
-            D(b, t): k * a**2,
+            Derivative(a, t): - k * a**2,
```

Comment:
Yes, I moved its definition from `api` to `support` in a recent commit, where it belongs. But apparently I forgot to add the line `from symfit.core.support import D` to `.api` apparently, that is the real issue here.

I don't dislike Derivative perse for internal usage, but I feel like tests are an external script to the actual package. As such, they should use the public api as documented, and preferably not internal functions. Of course that is not always possible, but as a general philosophy I try to stick to it.